#goal definition
# semantic_label: semantic label of the dst point cloud
# dst: point cloud to be aligned with
string semantic_label
sensor_msgs/PointCloud dst
---
#result definition
# aligned_object: transformed CAD model
sensor_msgs/PointCloud aligned_object
---
#feedback
float32 percent_complete